dabplus-on-air-processing.spec

```diff
@@ -54,4 +56,4 @@ install *.service %{buildroot}%{_exec_prefix}/lib/systemd/system/
 %doc %{_gh_mk_lp_name}-%{_gh_mk_lp_ref}/README.md
 %doc %{_gh_mk_lp_name}-%{_gh_mk_lp_ref}/LICENSE
 %{_exec_prefix}/lib/liquidsoap/%{_ls_version}/process.liq
-%attr(550, -, -) %{_exec_prefix}/lib/systemd/system/*.service
+%attr(550, -, -) %{_unitdir}/odr-padenc@dabplus-on-air-processing.service.d/*.conf
```

Comment:
Since there's now an actual service unit (override), shouldn't there be at least a systemd reload on install/uninstall? You can probably use some of the [systemd scriptlets and macros](https://fedoraproject.org/wiki/Packaging:Scriptlets?rd=Packaging:ScriptletSnippets#Systemd).